lib/olive_branch/middleware.rb

```diff
@@ -71,6 +78,14 @@ def call(env)
 
     private
 
+    def exclude_params?(env)
+      !@content_type_check.call(env["CONTENT_TYPE"]) || @exclude_params.call(env)
+    end
+
+    def exclude_response?(env, headers)
+      !@content_type_check.call(headers["Content-Type"]) || @exclude_response.call(env)
```

Comment:
organizational preference here would be to break the content type checks into separate methods. conceptually i think there's a disconnect between excluding params/response and checking for applicable content types. would probably go with something like this up top:
```ruby
if inflection && valid_content_type(env["CONTENT_TYPE"]) && !exclude_params(env)
``
makes the conditional longer but i think reads better